packages/rx_bloc_test/README.md

```diff
@@ -40,4 +42,54 @@ rxBlocTest<DetailsBloc,String>(
   skip: 3, // This will skip first two values + the initial value
   expect: ['Hello world'],
 )
-```
+```  
+
+## rxBlocFakeAsyncTest
+For the most complex test cases with multiple events, which contain `throttleTime` or/and `debounceTime` or similar inside, and to speed up test execution,`rxBlocFakeAsyncTest` should be used. It contains an instance of [FakeAsync](https://pub.dev/packages/fake_async) inside `act`, which provides a way to fire all asynchronous events that are scheduled for that time period without actually needing the test to wait for real time to elapse.
+
+**BAD:**
+```dart
+rxBlocTest<TimelineBloc, PaginatedList<SnapshotModel>>(
```

Comment:
It will be great to use the same example as above (`CounterBloc` ) covering the FakeAsync so that the people don't need to switch context while reading the README.md file